test/AutoRest.TestServer.Tests/lro.cs

```diff
@@ -104,7 +104,7 @@ public class LroTest: TestServerTestBase
         });
 
         [Test]
-        [Ignore("Handle multiple responses: https://github.com/Azure/autorest.csharp/issues/413")]
+        [Ignore("Handle empty response body: https://github.com/Azure/autorest.csharp/issues/300")]
```

Comment:
These paths are defined to return `Product`. However, these routes on the test server don't return a body.

Reply:
Is it supposed to be an invalid response? Should we fix a test server? How do other languages handle this test?

Reply:
It isn't supposed to be invalid. But, there are other tests that return empty bodies... which we haven't handled yet. I don't know what the recommendation is.

Reply:
I verified that empty response is invalid when swagger defines a model, so Assert.Throws should be fine

Reply:
Fixed.